### ISSUE TITLE
brand new install, no extensions appear

### ISSUE THREAD

**Issue by Darkvramp** [Jays2Kings/tachiyomiJ2K#1713]
### Steps to reproduce

Install on new phone
Install current version
Unable so see any extension downloads
![Screenshot_20240110-181316](https://github.com/Jays2Kings/tachiyomiJ2K/assets/48525649/497d6464-3870-4468-86a9-e7ef481e8beb)

### Expected behavior

Be able to see library of extensions 

### Actual behavior

No extensions library

### TachiyomiJ2K version

App version: 1.7.3

### Android version

Android 13

### Device

moto g power 5G - 2023

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I am reporting an issue exclusive to this fork. I have also checked that is not an issue on the [main version of Tachiyomi](https://github.com/tachiyomiorg/tachiyomi)
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[1.7.3](https://github.com/Jays2Kings/tachiyomiJ2K/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have admitted that I am a clown by having checked this box, as I have not read these acknowledgements.
- [X] I have filled out all of the requested information in this form.

**Comment by SusStew** [comment]
[Tachiyomi removed all extensions from its list](https://tachiyomi.org/news/2024-01-09-extensions-removal) and all extensions will need to be installed manually. Either use base Tachiyomi or a fork with extension repo support, install extension APKs manually from another repo, or wait until this fork adds support. Also see #1710.

(btw @dev pls fix, this has workarounds but built-in way would be more convenient)

**Comment by Darkvramp** [comment]
So there is no message whatsoever, inapp that the app is a broken piece of worthless crap now  the whole point of the app, was to have those extensions there and use them. Now that they are gone the app and this project is entirely defuct.

**Comment by Cryos-Kardia** [comment]
Unfortunately this literally just happened a few days ago and things are still getting worked out and sorted with Tachi's whole community. Basically from what I understand, the studio behind Solo Leveling went berserk on Tachiyomi legally and they're trying to protect themselves from legal assault.

**Comment by StudentSerai1181201811** [comment]
> So there is no message whatsoever, inapp that the app is a broken piece of worthless crap now. The whole point of the app, was to have those extensions there and use them. Now that they are gone the app and this project is entirely defuct.

No? Like the other guy said, there are workarounds and this only happened recently so be patient.
If you want to do so as soon as possible, either A. install the apks directly from 3rd party repos or B. install another fork that has 3rd party repo support and add that repo to it and fetch and install updates there.
Because of how Tachiyomi works, most if not all extensions will work in every fork if it has multiple sources, so installing extensions from Base/SY/AZ will make it so that J2K also has the extension installed

**Comment by Nitrousoxide** [comment]
> So there is no message whatsoever, inapp that the app is a broken piece of worthless crap now. The whole point of the app, was to have those extensions there and use them. Now that they are gone the app and this project is entirely defuct.

chill out friendo.  This is an open source project people are working on for no profit out of their own good will.   As other stated, install another fork or manually install the extensions you want manually for now.

**Comment by 1205311764** [comment]
怎么样自己添加扩展器啊？

**Comment by RokeJulianLockhart** [comment]
> close, since there is now support for custom extension repos?

Not until documentation exists. There's no indication after installation of where to get extensions from.
